packages/account-v2/src/pages/ProofOfOwnership/ProofOfOwnership.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { useGetAccountStatus } from '@deriv/api-v2';
+import { DerivLightWaitingPoaIcon } from '@deriv/quill-icons';
+import { Button, Loader, Text } from '@deriv-com/ui';
+import { IconWithMessage } from 'src/components';
+import { AUTH_STATUS_CODES } from 'src/constants';
+import { TAuthStatusCodes } from 'src/types';
+
+export const ProofOfOwnership = () => {
+    const { data, isLoading } = useGetAccountStatus();
```

Comment:
instead of using data maybe we can name it into accountStatus to make the code more readable 🤔